src/modules/completeTask.js

```diff
@@ -0,0 +1,14 @@
+import Todos from './Todos.js';
+
+const completeTask = (index, completed) => {
+  const todos = new Todos();
+  todos.loadTasks();
+  const tasks = todos.allTodos;
+
+  const taskIndex = tasks.findIndex((task) => task.index === index);
+  tasks[taskIndex].completed = completed;
+  todos.allTodos = tasks;
+  todos.saveTasks();
```

Comment:
- [ ] Great job so far, after marking a task as completed and refreshing the page the local storage shows it's marked as completed but the checkbox status(checked) doesn't persist, Kindly fix that.

<img width="845" alt="image" src="https://user-images.githubusercontent.com/59416136/221296472-964e1215-08e8-4761-a028-d92ce1cf5bb4.png">